README.md

```diff
@@ -1,7 +1,7 @@
 # js-debug-companion
 
-A companion extension to [js-debug](https://github.com/microsoft/vscode-js-debug) to enable remote Chrome debugging. You're probably don't want to instal this extension by itself, but for you interest, this is what it does.
+A companion extension to [js-debug](https://github.com/microsoft/vscode-js-debug) to enable remote Chrome debugging. You probably don't want to install this extension by itself, but for your interest, this is what it does.
 
-The scenario is if you are developing in a remote environment--like WSL, a container, ssh, or [VS Online](https://visualstudio.microsoft.com/services/visual-studio-online/)--and are port-forwarding a server to develop (and debug) in a browser locally. For remote development, VS Code runs two sets of extensions: one on the remote machine, and one on your local computer. `js-debug` is a "workspace" extension that runs on the remote machine, but we need to launch and talk to Chrome locally.
+The scenario is if you are developing in a remote environment—like WSL, a container, ssh, or [VS Codespaces](https://visualstudio.microsoft.com/services/visual-studio-codespaces/)—and are port-forwarding a server to develop (and debug) in a browser locally. For remote development, VS Code runs two sets of extensions: one on the remote machine, and one on your local computer. `js-debug` is a "workspace" extension that runs on the remote machine, but we need to launch and talk to Chrome locally.
```

Comment:
Personally I would leave the styling of em-dashes up to the renderer, but this is super nitpicky and Github doesn't em-dash correctly 😄

Reply:
Would you prefer the em-dashes change to be reverted? Can do too 🙂

Reply:
It's good as-is :)